modules/ding_nodelist/templates/ding_nodelist_widget_node_blocks.tpl.php

```diff
@@ -17,6 +17,7 @@
           'conf' => $conf,
           'row' => (int) ($i / 3),
           'column' => $i % 3,
+          'ai_id' => $i,
```

Comment:
What's and `$ai_id`? I see that's a the element index, but what means `ai`?
Also seems $k is not used here, for foreach can be replace by a `for`. That way $i will become available without initializing it.

Reply:
Removed.